nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/ckan/CKANBackend.java

```diff
@@ -96,12 +93,25 @@ public void persist(String orgName, String pkgName, String resName, String recor
      */
     private String resourceLookupOrCreateDynamicFields(String orgName, String pkgName, String resName, String records,DCATMetadata dcatMetadata, boolean createDataStore)
             throws Exception {
+        String orgId;
         if (!cache.isCachedOrg(orgName)) {
```

Comment:
did you pay attention to what is going on in https://github.com/easy-global-market/fiware-draco/blob/master/nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/ckan/CKANCache.java#L141?

Reply:
should I use  `isCachedOrg` then and change the API request to check if the orgnaization exists or not?

Reply:
seems to make sense

Reply:
when changing the API request, I no longer have access to the packages in the organization. Therefore https://github.com/easy-global-market/fiware-draco/blob/a4662f421b88e6d68b82c68c73679ad4e29540f4/nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/ckan/CKANCache.java#L175  doesn't work. Is it best to delete this part, knowing that later when creating the package the same check will be done and it will be cached anyway?

Reply:
yes, better delete it. currently, it is sort of hidden feature, not good.